pydm/tests/widgets/test_spinbox.py

```diff
@@ -82,49 +90,70 @@ def test_key_press_event(qtbot, signals, monkeypatch, first_key_pressed, second_
     signals.new_value_signal[float].emit(INIT_SPINBOX_VALUE)
 
     pydm_spinbox.setFocus()
+
     def wait_focus():
         return pydm_spinbox.hasFocus()
     qtbot.waitUntil(wait_focus, timeout=5000)
 
-    def press_key_and_verify(key_pressed, key_mod, key_press_count, expected_exp, expected_value):
+    def press_key_and_verify(
+            key_pressed, key_mod, key_press_count, expected_exp,
+            expected_value):
         if key_mod != Qt.NoModifier:
-            # Monkeypatch the Control flag because in this test, we don't properly have the app QApplication instance
-            monkeypatch.setattr(QApplication, "queryKeyboardModifiers", lambda *args: key_mod)
+            # Monkeypatch the Control flag because in this test, we don't
+            # properly have the app QApplication instance
```

Comment:
You can have the qapp by just adding it to the list of parameters by the side of the qtbot... it is a fixture of PyTest...

Reply:
Hi @hhslepicka. I just fixed the style here splitting the comment in two lines... I didn't change its content.

Reply:
@hhslepicka, can I mark this one as resolved?